code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -289,24 +289,41 @@
 	if (modified)
 		icon_state = "foamdart_empty"
 		desc = "Its nerf or nothing! ... Although, this one doesn't look too safe."
+		if(BB)
+			BB.icon_state = "foamdart_empty"
+	else
+		icon_state = "foamdart"
+		desc = "Its nerf or nothing! Ages 8 and up."
```

Comment:
I think you mean trademark. If you're going to kneejerk shut down ideas based on intellectual property law, then at least use the correct terminology.